GravatarApp/DesignSystem.swift

```diff
@@ -28,6 +34,24 @@ struct ActionButtonStyle: ButtonStyle {
     enum Style {
         case primary
         case secondary
+
+        var backgroundColor: Color {
+            switch self {
+            case .primary:
+                Color(uiColor: UIColor(light: .black, dark: .DS.almostWhite))
```

Comment:
Trying this almost white color for the dark mode. I can revert if this doesn't look good to you. @etoledom  To me it's a little better than full white.

Reply:
Looks great! We could call it  `contrastingBackgroundColor` or `oppositeBackgroundColor`, to differentiate it from the normal background color.